NextcloudTalk/DetailedOptionsSelectorTableViewController.m

```diff
@@ -87,7 +99,16 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
 {
     DetailedOption *option = [_options objectAtIndex:indexPath.row];
     UITableViewCell *cell = [[UITableViewCell alloc] initWithStyle:UITableViewCellStyleSubtitle reuseIdentifier:@"DetailOptionIdentifier"];
-    
+
+    if (_type == DetailedOptionsSelectorTypeAccounts) {
+        AccountTableViewCell *accountCell = [tableView dequeueReusableCellWithIdentifier:@"AccountTableViewCellIdentifier" forIndexPath:indexPath];
+        accountCell.accountNameLabel.text = option.title;
+        accountCell.accountServerLabel.text = [option.subtitle stringByReplacingOccurrencesOfString:@"https://" withString:@""];
```

Comment:
Should be case sensitive and should also cover `http://` ?

Reply:
It's possible when using a IP instead of FQDN: https://github.com/nextcloud/talk-ios/issues/252#issuecomment-1158975761
But I agree, it's probably an edge case...